e2e/discussions.spec.ts

```diff
@@ -173,7 +147,11 @@ test.describe('Discussions', () => {
       const threadReplyEdited = 'An edited reply to the Thread'
 
       await testAuth.step('First user goes to Project', async () => {
-        await userPage.goto(getDefaultDiscussionPath(userProject.id))
+        await goToDashboardPage(
+          userPage,
+          getDefaultDiscussionPath(userProject.id),
+        )
+
```

Comment:
Playwright's `goto` method waits for the 'load' event, which may be fired before SWC is ready for interaction. Instead, use custom `goTo***` methods that wait for SWC to fully load, e.g. SynapseNavDrawer is loaded for authenticated users.